tensorflow/python/framework/function_test.py

```diff
@@ -765,8 +766,12 @@ def Foo(x, y, z):
     # We added more randomness to function names in C API.
     # TODO(iga): Remove this if statement when we switch to C API.
     if ops._USE_C_API:  # pylint: disable=protected-access
-      self.assertEqual("Foo_aCYSbwBkR5A",
-                       Foo.instantiate([dtypes.float32] * 3).name)
+      if platform.machine() == "s390x":
```

Comment:
Since base64 output should be the same for all big endian machines, lets make this more generic and check for " sys.byteorder == 'big' " instead.